Sources/Fuzzilli/Core/ProgramBuilder.swift

```diff
@@ -1301,6 +1301,30 @@ public class ProgramBuilder {
         perform(EndIf())
     }
 
+    public struct SwitchBuilder {
+        public typealias SwitchCaseGenerator = () -> ()
+        fileprivate var defaultCaseGenerator: SwitchCaseGenerator? = nil
+        fileprivate var caseGenerators: [(value: Variable, fallsthrough: Bool, body: SwitchCaseGenerator)] = []
+
+        public mutating func addDefault(_ generator: @escaping SwitchCaseGenerator) { defaultCaseGenerator = generator }
+        public mutating func add(_ v: Variable, fallsThrough: Bool = false, body: @escaping SwitchCaseGenerator) {
+            caseGenerators.append((v, fallsThrough, body))
```

Comment:
Let's add a `assert(defaultCaseGenerator != nil)` here with a comment like "Default case must be generated first due to the way FuzzIL represents switch statements".